### ISSUE TITLE
reparametrization_net, a_u and a_v 

### ISSUE THREAD

**Issue by zhangchuanyi96** [shengliu66/SOP#5]
Thank you for your nice work. In the process of following your work, I have a few questions about the code.

1. `reparametrization_net = None` in `train.py`. If it's useless, why not delete codes related to it? It is confusing.
2. In your paper, `Table A.1: Hyper-parameters for SOP on CIFAR-10/100, Clothing-1M and Webvision datasets`, I find `a_u` and `a_v` can be different. However, in `loss.py`, I cant find how to modify different  `a_u` and `a_v`. I only find `ratio_reg` in the .json file.
3. In `loss.py`, `prediction = torch.clamp(original_prediction + U_square - V_square.detach(), min=1e-4)`, while Class-balance regularizer uses this `prediction`. So `V_square ` is not regularized? Is this a mistake?
4. `a_u` and `a_v` vary among different datasets. If I want to apply the method on other datasets, how can I choose proper values for them?
5. why learning rate (`a_u` and `a_v` ）are fixed instead of decay?

**Comment by shengliu66** [comment]
Thanks for pointing this out, originally the a_u a_v have different lr, this was achieved equivalently by control the MSE loss strength, but in order to make the code more consistent with the paper, I updated the code. Please let me know if you have any questions.
